src/Serilog.Sinks.EventLog/LoggerConfigurationEventLogExtensions.cs

```diff
@@ -55,7 +56,8 @@ public static LoggerConfiguration EventLog(
         string outputTemplate = DefaultOutputTemplate,
         IFormatProvider? formatProvider = null,
         LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
-        IEventIdProvider? eventIdProvider = null)
+        IEventIdProvider? eventIdProvider = null,
+        ICategoryNumberProvider? categoryNumberProvider = null)
```

Comment:
In the `EventLog.WriteEvent` API, the category is referred to as just `category` (rather than `categoryNumber`) - perhaps `ICategoryProvider`, or `IEventLogCategoryProvider` would be better?

Reply:
Renamed it to ICategoryProvider.